### PR TITLE
feat: Show whale collected nfts from particular creator

### LINKED ISSUE #1390: Show whale collected nfts from particular creator
When whale lands on creator profile. Show them somewhere in header how much nfts are collected from particular creator
Inspiration from Spotify

Enjoy chiptune hardbass

### PR DESCRIPTION
**Pull request by Jarsen136**
**Thank you for your contribution** to the [KodaDot NFT gallery](https://kodadot.xyz).
👇 \_ Let's make a quick check before the contribution.

### PR type

- [ ] Bugfix
- [x] Feature
- [ ] Refactoring

### What's new?

- [x] PR closes #1390
- [ ] <brief_description_of_what_I've_added>

### Before submitting Pull Request, please make sure:

- [x] My contribution builds **clean without any errors or warnings**
- [x] I've merged recent default branch -- **main** and I've no conflicts
- [x] I've tried to respect high code quality standards
- [x] I've didn't break any original functionality
- [x] I've posted a screenshot of demonstrated change in this PR

### Optional

- [ ] I've tested it at </rmrk/collection/26902bc2f7c20c546a-1FVG7>
- [ ] I've tested PR on mobile and everything seems works
- [ ] I found edge cases
- [ ] I've written some unit tests 🧪

### Had issue bounty label?

- [x] Fill up your KSM address: [Payout](https://beta.kodadot.xyz/transfer/?target=Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)

### Community participation

- [x] [Are you at KodaDot Discord?](https://discord.gg/35hzy2dXXh)

### Screenshot

- [x] My fix has changed **something** on UI; a screenshot is best to understand changes for others.

image

### CODE REVIEW

File: components/rmrk/Profile/ProfileDetail.vue [hunk 1]
```diff
@@ -313,6 +318,7 @@ export default class Profile extends mixins(
 
   public async mounted() {
     await this.fetchProfile()
+    this.fetchMyNftByIssuer()
```
Comment (kkukelka) [inline components/rmrk/Profile/ProfileDetail.vue:321]:
```suggestion
    await this.fetchMyNftByIssuer()
```

File: langDir/en.json [hunk 2]
```diff
@@ -520,7 +520,8 @@
     "sold": "Sold",
     "holdings": "Holdings",
     "gains": "Gains",
-    "showQr": "Show QR"
+    "showQr": "Show QR",
+    "collectedFromCreator": "I have {0} NFT(s) from this creator"
```
Comment (kkukelka) [inline langDir/en.json:524]: How do you like this?
```suggestion
    "collectedFromCreator": "Collected {0} NFT(s) from this creator"
```

**Review by kkukelka — CHANGES_REQUESTED** [review]
2 small comments. Let me know what you think 🤔

File: components/rmrk/Profile/ProfileDetail.vue (hunk 1 above)
Comment (Jarsen136) [reply components/rmrk/Profile/ProfileDetail.vue:321]: I wonder why we need this `await`. Seems that there is no any function need to be called after this `this.fetchMyNftByIssuer()`

File: langDir/en.json (hunk 2 above)
Comment (Jarsen136) [reply langDir/en.json:524]: thanks for your idea

File: components/rmrk/Profile/ProfileDetail.vue (hunk 1 above)
Comment (kkukelka) [reply components/rmrk/Profile/ProfileDetail.vue:321]: good argumentation 👍

**Review by kkukelka — APPROVED** [review]

**Comment by kkukelka** [comment]
pay 100

**Comment by yangwao** [comment]
😍 Perfect, I’ve sent the payout 
💵 $100 @ 75.35 USD/KSM ~ 1.327 $KSM
🧗 [Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG](https://kusama.subscan.io/account/Caiv9TbPz68q5dC8EcHu5xKYPRnremimGzqmEejDFNpWWLG)
🔗 [0xd2cd1c2123b545ddd28613933ad80ec4405ed2fe41da3e1ab014770c5e93cc08](https://kusama.subscan.io/extrinsic/0xd2cd1c2123b545ddd28613933ad80ec4405ed2fe41da3e1ab014770c5e93cc08)

🪅 Let’s grab another issue and get rewarded! 
🪄 [github.com/kodadot/nft-gallery/issues](https://github.com/kodadot/nft-gallery/issues/)

**Pull request merged**